actions/run_action.go

```diff
@@ -1,3 +1,33 @@
+/*
+Run Action
+
+Allows to run any available command or script in target chroot or
```

Comment:
in the filesystem ? target chroot seems like sudden new jargon again

Reply:
done.
used that for years ;-) may be a kind of local jargon but definitely not a new one